lms/static/js/spec/edxnotes/plugins/accessibility_spec.js

```diff
@@ -37,7 +37,9 @@ define([
         });
 
         afterEach(function () {
-            _.invoke(Annotator._instances, 'destroy');
+            while (Annotator._instances.length > 0) {
```

Comment:
Note to reviewers: With the new version of underscore, apparently a copy is not made of the array to be iterated before invoking the method. That means "destroy" of Annotator._instances[1] fails because the array length is only 1 at that point (since destroy removes the annotator being destroyed from the array).